cobc/codegen.c

```diff
@@ -387,6 +389,21 @@ lookup_source (const char *p)
 	return source_id++;
 }
 
+static void lookup_java_call(const char *p) {
+    struct call_list *clp;
+
+    for (clp = call_cache; clp; clp = clp->next) {
```

Comment:
I think it's another list you need to manipulate in this function. Something like `call_java_cache`.